src/persistence.rs

```diff
@@ -83,19 +81,19 @@ impl GossipPersister {
 			}
 		}
 
-		// print log statement every 10,000 messages
-		let mut persistence_log_threshold = 10000;
+		// print log statement every minute
+		let mut latest_persistence_log = Instant::now();
 		let mut i = 0u32;
-		let mut server_sync_completion_sent = false;
 		let mut latest_graph_cache_time = Instant::now();
 		// TODO: it would be nice to have some sort of timeout here so after 10 seconds of
 		// inactivity, some sort of message could be broadcast signaling the activation of request
 		// processing
 		while let Some(gossip_message) = &self.gossip_persistence_receiver.recv().await {
 			i += 1; // count the persisted gossip messages
 
-			if i == 1 || i % persistence_log_threshold == 0 {
+			if latest_persistence_log.elapsed().as_secs() >= 60 {
```

Comment:
it would be nice not to have to wait a minute to verify that persisting has started